File: src/core/java/src/net/lintim/model/ActivityType.java
package net.lintim.model;

/**
 * Enumeration of all possible activity types.
 */
public enum ActivityType {
    DRIVE("\"drive\""),
    WAIT("\"wait\""),
    CHANGE("\"change\""),
    TURNAROUND("\"turnaround\""),
    HEADWAY("\"headway\""),
    SYNC("\"sync\"");

    private String representation;

    ActivityType(String representation){
        this.representation = representation;
    }

    public String toString(){
        return representation;
    }
}

File: src/core/python/core/model/activityType.py
from enum import Enum


class ActivityType(Enum):
    """
    Enumeration of all possible activity types.
    """
    DRIVE = "\"drive\""
    WAIT = "\"wait\""
    CHANGE = "\"change\""
    TURNAROUND = "\"turnaround\""
    HEADWAY = "\"headway\""
    SYNC = "\"sync\""
    VIRTUAL = "\"virtual\""
